src/env/ipa-client-add.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/bash
+
+set -e
+
+. "$(dirname $0)/logs.sh" || exit 1
+
+ADMIN_PASSWD="SECret.123"
+USERNAME="ipa-user"
+DIR="/root/$USERNAME"
```

Comment:
yes, we can. but this is the default option, that is always rewritten with a value `card_dir` from the configuration file. I used /root because it was faster for development. but yes, will fix this